File: Iniciante/1060.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    float A,B,C,D,E,F;
    float i = 0;


        scanf("%f %f %f %f %f %f", &A, &B, &C, &D, &E, &F);


        if (A > 0)
        {
            i = i + 1;
        }

        else i = i+0;

            if (B > 0)
        {
            i = i + 1;
        }

        else i = i+0;


        if (C > 0)
        {
            i = i + 1;
        }

        else i = i+0;


        if (D > 0)
        {
            i = i + 1;
        }

        else i = i+0;


        if (E > 0)
        {
            i = i + 1;
        }

        else i = i+0;


        if (F > 0)
        {
            i = i + 1;
        }

        else i = i+0;




    printf("%0.0f valores positivos\n", i);

    return 0;
}

File: Iniciante/C#/1060.cs
/*
beecrowd | 1060 - Números Positivos

Faça um programa que leia 6 valores. 
Estes valores serão somente negativos ou positivos (desconsidere os valores nulos). 
A seguir, mostre a quantidade de valores positivos digitados.

Entrada: Seis valores, negativos e/ou positivos.

Saída: Imprima uma mensagem dizendo quantos valores positivos foram lidos.
*/

using System;

class URI
{

    static void Main(string[] args)
    {
        int contador = 0;

        for (int i = 0; i < 6; i++)
        {
            double valor = double.Parse(Console.ReadLine());
            
            if (valor > 0)
            {
                contador = contador + 1;
            }
            else
            {
                continue;
            }  
        }
        
        Console.WriteLine("{0} valores positivos", contador);
    }
}
